src/OpenTelemetry.Collector.Dependencies/SqlClientCollectorOptions.cs

```diff
@@ -0,0 +1,42 @@
+﻿// <copyright file="SqlClientCollectorOptions.cs" company="OpenTelemetry Authors">
+// Copyright 2018, OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// </copyright>
+using System.Data;
+
+namespace OpenTelemetry.Collector.Dependencies
+{
+    /// <summary>
+    /// Options for <see cref="SqlClientCollector"/>.
+    /// </summary>
+    public class SqlClientCollectorOptions
+    {
+        /// <summary>
+        /// Initializes a new instance of the <see cref="SqlClientCollectorOptions"/> class.
+        /// </summary>
+        public SqlClientCollectorOptions()
+        {
+        }
+
+        /// <summary>
+        /// Gets or sets a value indicating whether or not the <see cref="SqlClientCollector"/> should capture the text of a <see cref="CommandType.StoredProcedure"/> command. Default value: True.
+        /// </summary>
+        public bool CaptureStoredProcedureCommandContent { get; set; } = true;
```

Comment:
Name of the Stored Procedure more suiting than Text of a StoredProcedure?

Reply:
Updated